src/configRc.ts

```diff
@@ -14,6 +15,13 @@ export default class ConfigRc {
   public plugins: object[];
 
   constructor(plugins, argv) {
+    const finder = new FileFinder({
+      srcDir: process.cwd(),
+      exclude: ['node_modules', 'flow-typed', 'build'],
```

Comment:
Why specific dirs like `flow-typed` or `build` are ignored here? They are specific to the type of project. Spinjs is a generic tool

Reply:
Well... So far I've added them for testing purposes. I totally realize that project structure could be completely different.

Reply:
ok